apps/dashboard/app/javascript/stylesheets/editor.scss

```diff
@@ -44,17 +39,18 @@
 }
 
 .input-xs, select.input-xs {
-  font-size: 11px;
-  height: 20px;
-  padding-top: 0px;
+  font-size: 0.7em;
+  height: 2em;
+  padding-top: 0;
+  padding-bottom: 0;
 }
 
 .editor-navbar .control-label {
-  font-size: 11px;
+  font-size: 0.7em;
 }
 
 #save-button {
-  min-width: 80px;
+  background-color:rgb(2,117,216);
```

Comment:
I don't think a background color is viable. I use royal blue, and it looks kind of odd.

Maybe a border of the same color of the text (in this case white)?

![image](https://user-images.githubusercontent.com/4874123/179031677-ce7cba9c-8001-4a08-b9b6-bf228e56941b.png)